### DISCUSSION TITLE
Windows find '.netrc' file instead of '_netrc'

### DISCUSSION THREAD

**Discussion by Gamshik** [community/community#146478] [Programming Help]
### Body

I have  a problem with netrc file. I use this file for auth, my OS - windows, by default windows must be find a '_netrc' file, but in my case he is try find '.netrc', why this may happend???

18:46:01.406063 http.c:938              == Info: Couldn't find host github.com in the .netrc file; using defaults 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hey @Gamshik! Thanks for reaching out. I'm commenting so people can see your post and, hopefully, provide the help you need. Best of luck!

**Comment by Bdemirbelul** [comment]
On Windows, Git clients and other tools (like curl or git-credential) are supposed to look for a file named _netrc in your user directory (e.g., C:\Users\<username>\_netrc)   you can rename it or force it to do that by 

setx CURL_NETRC_FILE "C:\Users\<username>\_netrc"
